rllib/policy/torch_policy_v2.py

```diff
@@ -1022,7 +1048,28 @@ def _compute_action_helper(
         if self.model:
             self.model.eval()
 
-        if is_overridden(self.action_sampler_fn):
+        extra_fetches = {}
+        if isinstance(self.model, RLModule):
+            sample_batch = input_dict
+            if state_batches:
+                sample_batch.update({"state": state_batches})
```

Comment:
Can we add a new key to SampleBatch:
`INTERNAL_NN_STATES`? (and use this key here?)

This way, it's super clear. There is a lot of confusion as to state vs observation vs internal RNN states, etc.. going on.

Reply:
Ok to leave `state_batches` arg's name for now as we are still on the old Policy API (we should change this as we move on to the new classes).

Reply:
ack.

Reply:
Recurrent_states might be more apt